deployments/common/crds/appprotectdos.f5.com_dosprotectedresources.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
+  name: dosprotectedresources.appprotectdos.f5.com
+spec:
+  group: appprotectdos.f5.com
+  names:
+    kind: DosProtectedResource
+    listKind: DosProtectedResourceList
+    plural: dosprotectedresources
+    shortNames:
+      - pr
+    singular: dosprotectedresource
+  scope: Namespaced
+  versions:
+    - name: v1beta1
+      schema:
+        openAPIV3Schema:
+          description: 'DosProtectedResource defines a collection of Dos protected resources. status: preview'
```

Comment:
Why does DosProtectedResource (singular) define a collection (plural) of protected resources?
Why the preview flag?

Reply:
Updated to plural and preview removed.